### ISSUE TITLE
Run AutoDefine_oxfordlearnersdictionaries on multiple cards at once

### ISSUE THREAD

**Issue by artyompetrov** [artyompetrov/AutoDefine_oxfordlearnersdictionaries#7]
Hello Artem,  

Thank you for Anki plug in, I just love it!

I try to automate the card creation and import cards from a long text file.
Is it possible to run your plugin on multiple cards at once ? I mean, to select multiple rows from the table in the browser screen and then select the multiple rows and run the plug in on all rows (just like AwesomeTTS or google translate) ?  

Thanks,
Elad

**Comment by shrimansoft** [comment]
Yes, that will help add definitions to the word imported in bulk.

**Comment by artyompetrov** [comment]
added new option to Browse Edit menu 

![image](https://github.com/artyompetrov/AutoDefine_oxfordlearnersdictionaries/assets/31510247/710e4e2d-9859-4f61-8004-58636c556b24)

Only works with AutoDefineOxfordLearnersDictionary note type, make shure your card is converted to that note type before using that option.
